src/reason_pprint_ast.ml

```diff
@@ -2218,10 +2218,9 @@ let formatAttachmentApplication finalWrapping (attachTo: (bool * layoutNode) opt
           | (hd::tl, Some (useSpace, toThis)) ->
             (* TODO: Can't attach location to this - maybe rewrite anyways *)
             let attachedArgs = makeAppList attachedList in
-            label
-              ~space:true
-              (label ~space:useSpace toThis attachedArgs)
-              wrappedListy
+              (label ~space:useSpace toThis (label
```

Comment:
Some context: `wrappedListy` is for when we apply the special formatting logic I mentioned, to detect that a final function argument is list-like in nature, and therefore can "dock" to the entire binding etc (which matches common calling convention):

```rust
let x = myFunction {

};
```